src/components/HeaderContainer/HeaderContainer.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import SearchContainer from '../SearchContainer/SearchContainer';
+import MovieCard from '../MovieCard/MovieCard';
+
+import './HeaderContainer.scss';
+
+const HeaderContainer = (props) => {
+    const {
+        movieData,
+        handleSearchValue,
+        toggleSearch,
+        searchOption,
+    } = props;
+
+    return (
+        <div className="header-container">
+            <div className="buttons-container">
+                <a className="home-page-btn" href="."><b>netflix</b>roulette</a>
+                {movieData && <a className="home-page-btn" href=".">Search icon</a>}
+            </div>
+            {movieData ?
+                <MovieCard movieData={movieData} /> :
+                <SearchContainer
+                    handleSearchValue={handleSearchValue}
+                    toggleSearch={toggleSearch}
+                    searchOption={searchOption} />
+            }
+        </div>
+    );
+};
+HeaderContainer.defaultProps = {
```

Comment:
it's the best practice to create `defaultProps` and `propTypes` variables in the top of the file and then assign them to the component at the bottom